app/models/katello/pool.rb

```diff
@@ -26,6 +26,7 @@ class Pool < Katello::Model
     scoped_search :on => :consumed, :complete_value => true
     scoped_search :on => :account_number, :complete_value => true, :rename => :account
     scoped_search :on => :contract_number, :complete_value => true, :rename => :contract
+    scoped_search :on => :virt_who, :complete_value => true
```

Comment:
should this be only explicit search? (for search performance)
I guess the same is true for other search attributes.